### ISSUE TITLE
Deauthenticated session hangs

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#942]
If session data is invalid, it should automatically show a qr code and not just sit there.

**Comment by matheusm821** [comment]
if the status is UNPAIRED must delete data.json

**Comment by smashah** [comment]
@matheusm821 not ideally. The session data should be validated on launch. Also, UNPAIRED is actually not being fired anymore. I will be making a patch that will fire `DISCONNECTED` instead. This issue and the fact that `UNPAIRED` are not being fired are a result of bugs on WA's end.

**Comment by smashah** [comment]
2bc13ad53db1c1743e5c29cf7a7c689feb00cc32

**Comment by smashah** [comment]
The patch fixes UNPAIRED. `UNPAIRED` is now manually being fired when `DISCONNECTED` is detected.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.1.5](https://github.com/open-wa/wa-automate-nodejs/compare/2.1.4...2.1.5) (2020-10-16)

- Deauthentication issues resolved #942 [`#942`](https://github.com/open-wa/wa-automate-nodejs/issues/942)
- Release 2.1.5 [`04ad893`](https://github.com/open-wa/wa-automate-nodejs/commit/04ad893ffcc0f0f4f62a89acb4777df0ef843791)
- Patch: `onStateChanged` now fires `DISCONNECTED`. `UNPAIRED` not working due to WA bug. [`2bc13ad`](https://github.com/open-wa/wa-automate-nodejs/commit/2bc13ad53db1c1743e5c29cf7a7c689feb00cc32)
- Update patches [`7d73a08`](https://github.com/open-wa/wa-automate-nodejs/commit/7d73a088825b739a04f6bab113fe4171aa99cfd3)
